README.md

```diff
@@ -1,16 +1,204 @@
+            <pre class="line-numbers"><code class="prism language-php" data-prism>//php code here..</code></pre>
+        </div>
+    </li>
+</ul>
+```
+
+For multi-line code examples, the code must be pasted in verbatim between the `<code>` tags.
+
+The following will render correctly:
+```html
+<div class="ide">
+    <pre class="line-numbers"><code class="prism language-http" data-prism>HTTP/1.1 429
+Date: Tue, 20 Aug 2013 14:50:41 GMT
+Status: 429
+X-RateLimit-Limit: 60
+X-RateLimit-Remaining: 0
+X-RateLimit-Reset: 1377013266
+{
+    "message": "Too many requests",
+    "code": 429
+}</code></pre>
+</div>
+```
+
+The following will have unnecessary line-breaks and indentations. 
+```html
+<div class="ide">
+    <pre class="line-numbers"><code class="prism language-http" data-prism>
+        HTTP/1.1 429
+        Date: Tue, 20 Aug 2013 14:50:41 GMT
+        Status: 429
+        X-RateLimit-Limit: 60
+        X-RateLimit-Remaining: 0
+        X-RateLimit-Reset: 1377013266
+        {
+            "message": "Too many requests",
+            "code": 429
+        }
+    </code></pre>
+</div>
+```
+
+> Don't forget to use proper indenting for all code examples. The indenting of the code examples should be independent from the indentation of the surrounding HTML tags.
+
+For referencing code inline, especially when the code is a non-executable snippet, use spans with `class=tag`. For example, use the method <span class="tag">listDocuments()</span>.
+
+
+
+#### Placeholder Values
+- Be consistent with placeholder values for name type fields across languages. For example, all examples for the Databases query guide should use the database `catalogue` and collection `movies`.
+- Where applicable, use `"unique()"` for ids that can be auto-generated.
+- Use a concise value like `"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ..."` for hash values like long IDs and API keys.
+- Use a placeholder for hostnames like `"https://[HOSTNAME_OR_IP]/v1"` in code examples, but be consistent.
+- Use a known generic name as a place holder like `"John Doe"`, avoid real name and celeberty names.
+- Use generic email passwords as placeholders like `"email@example.com", "password"`.
+- Use a relative file name that is context appropriate like `"img/file.png"`.
+- Where possible, use names for teams, collections, databases, buckets, and other resources that are contextually appropriate and hint at the resource type. This is especially true for code examples in guides like `const result = storage.getFilePreview('image_bucket', 'smiling_penguine.png', '240px', '180px');`, where the parameters passed in are not described.
+- Use sensible variable names :)
+
 ### Notices
+Use notices to point out important information, especially those relevant for first time readers. 
+
+Use the following HTML structure to add important notes inside your docs:
+
+```html
+<div class="notice">
+  <h2>Important Message</h2>
+  <p>Message content here.</p>
+</div>
+```
 
 ### Screenshots
```

Comment:
Reading this section, we say:
- If no dark mode is provided, light mode will be the fallback
- All screenshots of the Appwrite dashboard should support light and dark mode
- All screenshot of the Appwrite dashboard should support light and dark mode.

We say it too much. Just making sure we are aware and do it in purpose.

Reply:
Good catch